pennylane/templates/subroutines/select.py

```diff
@@ -98,6 +99,21 @@ def __init__(self, ops, control, id=None):
         all_wires = target_wires + control
         super().__init__(*self.data, wires=all_wires, id=id)
 
+    def __copy__(self):
+        """Copy this op"""
+        cls = self.__class__
+        copied_op = cls.__new__(cls)
+
+        new_data = copy.copy(self.data)
```

Comment:
Are we sure we don't need `deepcopy` here?

Reply:
I don't think so. This is the same as in the `Operator` class.